### PR TITLE
Improve documentation

### PR DESCRIPTION
**Pull request by andysellick**
## What
Minor tweaks to some of our documentation.

## Why
Some areas of our documentation were not as clear as they could be.

## Visual Changes
Changes to docs only.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,17 +37,30 @@ yarn run jasmine:ci
 
 ### Further documentation
 
+Using the gem
```
Comment (injms) [inline README.md:40]: Do you think that this should be a heading?

```suggestion
#### Using the gem
```

File: docs/auditing.md [hunk 2]
```diff
@@ -2,6 +2,8 @@
 
 The component guide includes some auditing of components and applications that can provide useful information. It can be found at `/component-guide/audit`.
 
+It is designed to be run locally as it gets information about applications and how they use components from local files. Note that if your local copy of an application is not up to date the information reported by the auditing may be incorrect.
```
Comment (injms) [inline docs/auditing.md:5]: I think that the local apps need to be up to date is worth highlighting in a way that can't be missed - not necessarily as I've suggested, just in any way that highlights it strongly:

```suggestion
It is designed to be run locally as it gets information about applications and how they use components from local files. 

> **If your local copy of an application is not up to date the information reported by the auditing may be incorrect**
```

File: README.md (hunk 1 above)
Comment (injms) [reply README.md:40]: Or add a colon at the end:
```suggestion
Using the gem:
```

Comment (andysellick) [reply README.md]: Have updated the headings here a bit based on this suggestion, hopefully a bit clearer now.

File: docs/auditing.md (hunk 2 above)
Comment (andysellick) [reply docs/auditing.md]: Thanks, have highlighted this (I couldn't think of a better way than you'd suggested, it's clear at least).

**Review by injms — APPROVED** [review]
Smashing - thanks!

**Pull request merged**